client/views/teams/CreateTeamModal/CreateTeamModal.tsx

```diff
@@ -338,7 +334,7 @@ const CreateTeamModal: FC<CreateTeamModalProps> = ({ onClose }) => {
 							({t('optional')})
 						</Box>
 					</Field.Label>
-					<UsersInput value={members} onChange={onChangeMembers} />
+					<UserAutoCompleteMultiple handleById value={members} onChange={onChangeMembers} />
```

Comment:
Please, try to get rid of `handleById` prop. I believe it's achievable by using a custom callback signature for `onChange` (e.g. `onChange(username, uid)`).